tests/src/test/javascript/crypto/EccCryptoContextSuite.js

```diff
@@ -0,0 +1,245 @@
+/**
```

Comment:
The Java test generates keys on the fly, just as the RSA javascript and Java tests do.  For the new ECC javascript test I chose to import hard-coded test vectors in JWK format.  The Java test could be adapted to use the same.

I saw a number of places in the RSA test vectors that referenced 'TODO/FIXME' in regards to loading pre-defined test keys - perhaps updating the Java ECC test could be part of that effort.